src/usr/local/www/system_camanager.php

```diff
@@ -616,6 +688,11 @@
 	$pconfig['caref'],
 	$allCas
 ));
+$group->add(new Form_Input(
```

Comment:
Should this be in a separate section? I couldn't find a way to make it look nice when both are toggled, so I added the password field to the group.

Reply:
Have to check it out when testing to be sure but it's probably OK like this

Reply:
👍 It looks fine on desktop, it even flows nicely on small screens.